src/components/feedback/Snackbar/index.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import styles from './styles.less';
+import classNames from 'classnames';
+
+export enum verticalPositions {
+    top = 'top',
+    bottom = 'bottom',
+}
+export enum horizontalPositions {
+    left = 'left',
+    right = 'right',
+    center = 'center',
+}
+
+type SnackBarProps = {
+    verticalPosition?: verticalPositions;
+    horizontalPosition?: horizontalPositions;
+    open?: boolean;
```

Comment:
Should the snackbar be open by default and then only close when the abort signal is aborted?